homeassistant/components/deconz/light.py

```diff
@@ -289,13 +289,17 @@ def async_update_callback(self) -> None:
 class DeconzGroup(DeconzBaseLight[Group]):
     """Representation of a deCONZ group."""
 
+    _attr_has_entity_name = True
+
     _device: Group
 
     def __init__(self, device: Group, gateway: DeconzGateway) -> None:
         """Set up group and create an unique id."""
         self._unique_id = f"{gateway.bridgeid}-{device.deconz_id}"
         super().__init__(device, gateway)
 
+        self._attr_name = ""
```

Comment:
Why is this `""`?

Reply:
Because its given self._attr_name = device.name in the parent class

Reply:
Right, but shouldn't it be set to `None` in that case?

Reply:
Aah yes perhaps :)

Reply:
Changed